Meme-ify-Me/app/src/main/java/nyc/c4q/rosmaryfc/meme_ify_me/VanillaMemeEdit.java

```diff
@@ -22,6 +38,40 @@ public boolean onCreateOptionsMenu(Menu menu) {
         return true;
     }
 
+    public void onShareClick(View v){
```

Comment:
I moved the share onclick method from main activity.java to the vanilla meme edit. because the share button was moved from main activity to vanilla meme